splinterd/man/splinterd.1.md

```diff
@@ -185,12 +185,19 @@ OPTIONS
 `--peers PEER-URL` `[,...]`
 : Specifies one or more Splinter nodes that `splinterd` will automatically
   connect to when it starts. The *PEER-URL* argument must specify another node's
-  network endpoint, using the format `protocol_prefix://ip:port`.
+  network endpoint, using the format `protocol_prefix://ip:port` or
```

Comment:
This should probably document the default authorization being challenge.

Reply:
Added